cmd/cdi-importer/importer.go

```diff
@@ -135,6 +135,21 @@ func main() {
 		touchDoneFile()
 		preallocationApplied = processor.PreallocationApplied()
 	}
+	// With writeback cache mode it's possible that the process will exit before all writes have been commited to storage.
+	// To guarantee that our write was commited to storage, we make a fsync syscall and ensure success.
+	// Also might be a good idea to sync any chmod's we might have done.
+	dataFile := getImporterDestPath(contentType, volumeMode)
```

Comment:
I think It only applies for the branch of code which actually creates the file, and before the touchDoneFile() at line 134.

Reply:
there is a chmod also for the other branch of code:
https://github.com/kubevirt/containerized-data-importer/blob/main/pkg/image/qemu.go#L300

Reply:
you are right!